.rubocop.yml

```diff
@@ -29,3 +29,5 @@ Style/HashTransformValues:
   Enabled: false
 Rails/FindBy:
   Enabled: false
+Rails/ApplicationController:
```

Comment:
Can we skip this check only in `ErrorsController` and not globally?

Reply:
I've now removed this as I've now made the `ErrorsController` inherit from `ApplicationController`